### PR TITLE
In App voice tutorial & required set up guides for iOS

### PR DESCRIPTION
**Pull request by brittBarak**
## Description

1. Tutorial to get started with in app voice (simple 1:1 calls) with Client SDK on an iOS app.
2. a guide to add the SDK to an iOS app (to support the tutorial)
3. a guide to setup push notifications with the SDK on an iOS app. (quite basic and essential functionality for devs who get started with the SDK)

### CODE REVIEW

**Comment by tbedford** [comment]
I've added the `Don't merge` label back as I will do this one by hand when the material is ready.

File: _tutorials/client-sdk/get-started/in-app-voice/ios.md [hunk 1]
```diff
@@ -242,6 +255,8 @@ To read more about the permissions needed, [see here.](_documentation/client-sdk
 
 ---
 
-# Congratulations!
+#Congratulations!
```
Comment (tbedford) [inline _tutorials/client-sdk/get-started/in-app-voice/ios.md]: If you click on the "View File" button in GitHub you will see the issue. This should be second level heading and there should be a space after the two hash symbols.

File: _documentation/client-sdk/setup/add-sdk-to-your-app/ios.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+2. Under your target add the `NexmoClient` pod. Replace `TargetName` with your actual target name.
+
+   ```ruby
+   target 'TargetName' do
+       pod 'NexmoClient'
+   end
+   ```
+
+   * Make sure the pod file has the public CocoaPod specs repository source.
```
Comment (tbedford) [inline _documentation/client-sdk/setup/add-sdk-to-your-app/ios.md]: Number 3. appears missing - should this be 3?

File: _documentation/client-sdk/setup/add-sdk-to-your-app/ios.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+**Objective-C**
+```objective-c
+NXMClient *client = [NXMClient new];
+[client setDelegate:self];
+[client loginWithAuthToken:@"your token"];
+```
+
+> *Note*: self should implement `NXMClientDelegate` protocol.  
+
+On a succesfull login, The following delegate method is called with `isLoggedIn = true`.
```
Comment (tbedford) [inline _documentation/client-sdk/setup/add-sdk-to-your-app/ios.md]: "succesfull" -> successful.

File: _tutorials/client-sdk/get-started/in-app-voice/android.md [hunk 4]
```diff
@@ -119,7 +119,7 @@ This will also allow you to start a phone (PSTN) call, by adding a phone number
 
 ## 3. Register to incoming events
 
-When Jane calls Joe, Joe should be notified about it, for example, in order to answer the call.
+When Jane calls Joe, Joe should be notified about it, for example, in order to answer or decline the call.
```
Comment (tbedford) [inline _tutorials/client-sdk/get-started/in-app-voice/android.md]: Wasn't this an iOS PR?

File: _tutorials/client-sdk/get-started/in-app-voice/ios.md [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+    - `#import <NexmoClient/NexmoClient.h>` - imports the sdk
+    - `@property NXMClient *nexmoClient;` - property for the client instance
+    - `@property NXMCall *ongoingCall;` - property for the call instance
+---
+
+## 1. Login
+Using the Nexmo SDK should start with a `NexmoClient` login, using a `jwt` user token.
+
+On production apps, your server would authenticate the user, and would return a `jwt` to the app.
+You can read more about generating the `jwt` [here]("https://developer.nexmo.com/stitch/concepts/jwt-acl").
```
Comment (tbedford) [inline _tutorials/client-sdk/get-started/in-app-voice/ios.md]: "/stitch/" -> "/client-sdk/"

File: _tutorials/client-sdk/get-started/in-app-voice/ios.md [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+
+For testing and getting started purposes, you can use the `jwt` generated for you on the dashboard.
```
Comment (tbedford) [inline _tutorials/client-sdk/get-started/in-app-voice/ios.md]: Whats Dashboard? Nexmo Dashboard?

Comment (brittBarak) [reply _tutorials/client-sdk/get-started/in-app-voice/ios.md]: we made a simple web page. but will hold with it a little. I added the CLI guide link instead.

**Comment by tbedford** [comment]
Content incorporated into PR 1210.

**Comment by tbedford** [comment]
@mheap : yes, I checked, although as far as I know there is not a public download for SDK as yet - I can add that when I know it.

**Pull request closed without merging**